### PR TITLE
updated "groups" for the ZFIN curators

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

**Comment by kltm** [comment]
@sabrinatoro Looks like non-YAML spacing around L2023.

File: metadata/users.yaml [hunk 1]
```diff
@@ -2017,7 +2020,10 @@
     noctua-go:
       allow-edit: true
   uri: 'http://orcid.org/0000-0002-2244-7917'
-  xref: 'GOC:cvs'
+ groups:
```
Comment (kltm) [inline metadata/users.yaml]: Spacing.

File: metadata/users.yaml [hunk 2]
```diff
@@ -2017,7 +2020,10 @@
...
+    - 'http://zfin.org'
+    - 'http://geneontology.org'
+ xref: 'GOC:cvs'
```
Comment (kltm) [inline metadata/users.yaml]: Spacing.

**Comment by sabrinatoro** [comment]
@kltm Thank you! 
I don't know how to get to the file I submitted so I can edit it and solve to this spacing issue. Could you point me in the right direction, please? 
(we can take this out of GitHub if this should not be discussed here)
Thanks!

**Comment by kltm** [comment]
@sabrinatoro Apparently, I had the ability to edit the branch directly, but not suggest a change on a fork of your fork in the interface. Weird.
Anyways, I think this is correct now.

**Comment by kltm** [comment]
I think this looks good--thank you!

**Comment by sabrinatoro** [comment]
Great! Thank you!

**Pull request merged**